x-pack/plugins/observability_solution/apm/server/routes/entities/get_entities.ts

```diff
@@ -72,7 +72,10 @@ export async function getEntities({
 
   return entities.map((entity): ServiceEntities => {
     return {
-      serviceName: entity.entity.identityFields.service.name,
+      serviceName: entity.service.name,
+      environment: Array.isArray(entity.service?.environment) // TODO fix this in the EEM
```

Comment:
Q: Do we need to support multiple environments? And how do we want to fix it - have a single environment or have an array (I see currently we get the first one in case of an array)?

Reply:
We support services with multiple environments already. In service inventory we group them.  This seems to be introduces by the recent changes in EEM. Which needs to be fixed in their schema. 

If the environment exists they return a string. If the environment is empty they return an empty array. This needs to be consistent. I need to create an issue

Reply:
https://github.com/elastic/kibana/issues/188820